.circleci/config.yml

```diff
@@ -73,7 +73,7 @@ commands:
             apt-get update && apt-get install -y nodejs
       - run:
           name: Install serverless CLI
-          command: npm i -g serverless
+          command: npm i -g serverless@^3
```

Comment:
I'm amazed this worked on dev. :O
Seems like serverless v3 didn't support `dotnet8` as far as on v3.38 [[link](https://forum.serverless.com/t/does-serverless-support-dotnet-8-0/19980/2)].

I mean, looking at the pipeline logs, it still gives an error message that dotnet8 is an invalid runtime:
```
Warning: Invalid configuration encountered
  at 'provider.runtime': must be equal to one of the allowed values [dotnet6, go1.x, java21, java17, java11, java8, java8.al2, nodejs14.x, nodejs16.x, nodejs18.x, nodejs20.x, provided, provided.al2, provided.al2023, python3.7, python3.8, python3.9, python3.10, python3.11, ruby2.7, ruby3.2]
```